### PR TITLE
Kinetis SDK 2.0

### PR DESCRIPTION
**Pull request by thinkberg**
Some actually minor changes to make the code compatible with Kinetis SDK 2.0. No efforts were made to keep compatibility with 1.3 though. But maybe it helps.

### CODE REVIEW

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

File: wolfcrypt/src/asn.c [hunk 1]
```diff
@@ -385,7 +385,7 @@ time_t ksdk_time(time_t* timer)
     if (timer == NULL)
         timer = &localTime;
 
-    *timer = (PIT_DRV_ReadTimerUs(PIT_INSTANCE, PIT_CHANNEL)) / 1000000;
+    *timer = COUNT_TO_USEC(PIT_GetCurrentTimerCount(PIT_INSTANCE, PIT_CHANNEL), SystemCoreClock) / 1000000;
```
Comment (thinkberg) [inline wolfcrypt/src/asn.c]: This line may be an issue, as the original code had no dependency on a specific clock. However, the old method is removed in KSDK 2.0 in favor of the COUNT_TO_USEC macro, so we have to include a clock here.

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by JacobBarthelmeh** [comment]
Hi thinkberg.

Thanks for submitting the pull request. We are looking into the changes. Is the email on the commit the best place to contact you in case of sending a contributor agreement for accepting the code changes?

Regards,
Jacob

**Comment by thinkberg** [comment]
Use this address: matthias.jugel@streamdrill.com

Thanks,

	Leo.
-- 
Matthias L. Jugel - ubirch GmbH
http://ubirch.com/

On 11 April 2016 at 23:34:47, JacobBarthelmeh (notifications@github.com) wrote:

Hi thinkberg.

Thanks for submitting the pull request. We are looking into the changes. Is the email on the commit the best place to contact you in case of sending a contributor agreement for accepting the code changes?

Regards,
Jacob

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by kaleb-himes** [comment]
Can one of the admins verify this patch?

**Comment by JacobBarthelmeh** [comment]
Hello thinkberg, thank you for opening the pull request. We are going to treat it as a feature request to update Kinetis SDK version with wolfSSL.

**Comment by thinkberg** [comment]
yes, fine with me.

**Pull request closed without merging**